code_annotations/find_django.py

```diff
@@ -75,7 +77,7 @@ def seed_safelist(self):
 
 """
             safelist_file.write(safelist_comment.lstrip())
-            yaml_ordered_dump(safelist_data, stream=safelist_file, default_flow_style=False)
+            yaml.safe_dump(safelist_data, stream=safelist_file, default_flow_style=False)
```

Comment:
I'm also just generally unclear as to why these don't need to be ordered anymore.

Reply:
Oddly sorted dicts seem to be the default behavior in pyyaml so this seems to have not been necessary in the first place: https://github.com/yaml/pyyaml/blob/4c2e993321ad29a02a61c4818f3cef9229219003/lib3/yaml/representer.py#L108-L114

```I checked this in Python 3.6 and 2.7:
>>> a = {'z':1, 'g':2, 'a': 3, 'b': 4, 'w': 5}
>>> import yaml
>>> yaml.safe_dump(a)
'{a: 3, b: 4, g: 2, w: 5, z: 1}\n'```

I found it because the new edx-lint enforces `yaml.safe_load/dump`, which fails on the old OrderedDict approach.